### DISCUSSION TITLE
Last plugin update for VS Code looks broken wrong checks for Flask and Session.

### DISCUSSION THREAD

**Discussion by KashaMalaga** [microsoft/pyright#5587] [General]
![image](https://github.com/microsoft/pyright/assets/879449/9bf6f14e-5f31-439d-bcfc-147a6e854045)
Your latest update its wrong, almost wrong warnings everywhere 👎

**Comment by erictraut** [comment]
@KashaMalaga, if you think there is a bug, please file a bug report and include repro steps and a minimal, self-contained code sample (in text form). I can't debug issues from partial screen shots.
